src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+
+  </body>
```

Comment:
Don't need empty line here.
```suggestion
    Hello, world!
  </body>
```

Reply:
Thanks